pkg/reexec/reexec.go

```diff
@@ -35,8 +35,14 @@ func Self() string {
 	name := os.Args[0]
 	if filepath.Base(name) == name {
 		if lp, err := exec.LookPath(name); err == nil {
-			name = lp
+			return lp
 		}
 	}
+	// handle conversion of relative paths to absolute
+	if absName, err := filepath.Abs(name); err == nil {
+		return filepath.Clean(absName)
```

Comment:
Ah yes; I had missed that when I was looking at that code earlier.  Updated push takes out the call to `Clean()`.  Thanks!